about/package-scope.md

```diff
@@ -30,9 +28,229 @@ open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSc
 one of our editors. We are happy to look at your package and help you understand
 whether it is in scope or not.
 
-```{include} /appendices/scope.md
+## About the scope of packages that we review
+
+pyOpenSci supports packages that support both open reproducible science,
+data processing and and the various stages of managing the
+data lifecycle. Packages submitted to pyOpenSci should fit into one or
+more of the categories below.
+
+```{admonition} Package Use Metrics Are Not a Requirement for Review
+:class: important
+pyOpenSci has no requirements surrounding the volume community use of your
+package.
+If your package is well developed, the number of GitHub stars or PyPI or conda
+downloads is NOT considered as a part of our scope evaluation.
+
+We welcome young packages that are just entering the scientific Python
+ecosystem to apply for review if they are relevant to the science community and
+fit into at least one scope category below.
 ```
 
+If you are unsure whether your package fits into one of the general or
+statistical categories, please open an issue as a pre-submission inquiry (Examples). <todo add link>
+
+```{note}
+This is a living document. The categories below may change through time.
+This may mean in some cases, some previously peer review-accepted packages
+may not be in-scope today. We strive for consistency in our peer review process. However, we also evaluate packages on a case-by-case basis.
+In some cases exceptions are made.
+```
+
```

Comment:
NOTE: i think we should add a policy around telemetry in packages. a current review has it. i think we should discourage against it but if it's required make it opt in rather than opt out preferred?

We should also evaluate whether we want to review general data manipulation and coding tools. ROS does not review these tools. but defining what that means is tricky.

Reply:
Perhaps start with a narrower scope? It's easy to broaden it later, but not the other way around.

Reply:
good point! is this a specific comment related to our scope as defined now or just to that statement above? I added that given Ropensci has similar language and it just allows us to adjust in the future in case we realize we are headed down a rabbit hole that doesn't work for us! (kind of like a legal disclaimer :) in case we need to modify scope)

We've had the same scope for the past 4 years BUT of course we are being more specific now about each category given all of the questions we've gotten :)